src/HttpHdrScTarget.h

```diff
@@ -30,11 +32,11 @@ class HttpHdrScTarget
     static const int MAX_AGE_UNSET=-1; //max-age is unset
     static const int MAX_STALE_UNSET=0; //max-stale is unset
 
-    HttpHdrScTarget(const char *target_):
+    explicit HttpHdrScTarget(const char *target_) :
         mask(0), max_age(MAX_AGE_UNSET), max_stale(MAX_STALE_UNSET),target(target_) {}
-    HttpHdrScTarget(const String &target_):
+    explicit HttpHdrScTarget(const String &target_) :
         mask(0), max_age(MAX_AGE_UNSET), max_stale(MAX_STALE_UNSET),target(target_) {}
-    HttpHdrScTarget(const HttpHdrScTarget &t):
+    explicit HttpHdrScTarget(const HttpHdrScTarget &t) :
         mask(t.mask), max_age(t.max_age), max_stale(t.max_stale),
         content_(t.content_), target(t.target) {}
```

Comment:
If the existing code copies HttpHdrScTargets, then please explicitly default this copy constructor (if possible):

```suggestion
    explicit HttpHdrScTarget(const HttpHdrScTarget &) = default;
```

If the existing code does not copy HttpHdrScTargets, then please delete this copy constructor:

```suggestion
    explicit HttpHdrScTarget(const HttpHdrScTarget &) = delete; // avoid accidental string copies
```

Reply:
Done in 8ed967f14

Reply:
Not done. I refreshed this old request at https://github.com/squid-cache/squid/pull/589#discussion_r454734913 and now resolving this outdated one.